src/renderers/shaders/ShaderChunk/bsdfs.glsl.js

```diff
@@ -268,17 +279,15 @@ vec3 BRDF_Specular_GGX_Environment( const in GeometricContext geometry, const in
 // http://www.jcgt.org/published/0008/01/03/
 void BRDF_Specular_Multiscattering_Environment( const in GeometricContext geometry, const in vec3 specularColor, const in float roughness, inout vec3 singleScatter, inout vec3 multiScatter ) {
 
-	float dotNV = saturate( dot( geometry.normal, geometry.viewDir ) );
+	float dotNV = max( 0.01, dot( geometry.normal, geometry.viewDir ) );
 
-	vec3 F = F_Schlick( specularColor, dotNV );
+	vec3 F = F_Schlick_RoughnessDependent( specularColor, dotNV, roughness );
```

Comment:
You are passing in `dotNV` to your method, but the method as written expects `dotLH`.

Reply:
The Fdez-Aguera paper uses the normal vector rather than the halfway vector. Not sure if that's required, or if this should use the halfway vector like other instances. I'll do some side by side tests.

Either way, as this is a new schlick function, the signature needs to be updated. Good catch!

Reply:
Updated